vimeo-networking/src/main/java/com/vimeo/networking/model/Video.java

```diff
@@ -213,6 +213,10 @@ public String toString() {
     @SerializedName("spatial")
     public Spatial mSpatial;
 
+    @Nullable
+    @SerializedName("live")
+    private Live mLive;
```

Comment:
I thought about that, but since it's the direction we want to go in, and it's a new field, I thought this would be better practice. How about me leaving the accessors and making the field public?